lib/rules/landmark-one-main.json

```diff
@@ -3,10 +3,10 @@
 	"selector": "html",
 	"tags": ["cat.semantics", "best-practice"],
 	"metadata": {
-		"description": "Ensures the document has only one main landmark and each iframe in the page has at most one main landmark",
+		"description": "Ensures the document has a main landmark and each iframe in the page has a main landmark",
```

Comment:
This has changed a little. This rule should check that the top window has at least one main landmark. What's in iframes shouldn't matter for this rule. Then the new rule should check that there is no more than one main in any given document. That way the top window must have 1 main, and iframes can have 0 or 1 main.